azure-pipelines.yml

```diff
@@ -59,6 +59,18 @@ stages:
         python3 setup.py test
       displayName: "Unit tests"
 
+    - script: |
+        set -ex
+        # Install .NET CORE
+        wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+        sudo dpkg -i packages-microsoft-prod.deb
+        sudo apt-get update
+        sudo apt --fix-broken install -y
```

Comment:
>broken [](http://example.com/codeflow?start=23&length=6)

What is broken and why?

Reply:
since there is error without fix-broken:
https://dev.azure.com/mssonic/build/_build/results?buildId=19146&view=logs&j=83516c17-6666-5250-abde-63983ce72a49&t=c10d5f44-55ce-55d7-7975-407ed75d9a96

Reply:
also for directlly install dot sdk:
https://dev.azure.com/mssonic/build/_build/results?buildId=19148&view=logs&j=83516c17-6666-5250-abde-63983ce72a49&t=c10d5f44-55ce-55d7-7975-407ed75d9a96

Reply:
It should be caused by wrong package packages-microsoft-prod installed, the script is run inside the container, so we need to install the package for debian buster, not ubuntu 20.04.
Please follow https://docs.microsoft.com/en-us/dotnet/core/install/linux-debian, and try again, thanks.

Reply:
right, have change to debian 10 and remove fix-broken!